code/drasil-code/test/VectorTest.hs

```diff
@@ -7,7 +7,7 @@ import GOOL.Drasil (GSProgram, SVariable, SMethod, OOProg, ProgramSym(..),
   ModuleSym(..))
 
 vectorTest :: OOProg r => GSProgram r
-vectorTest = prog "VectorTest" "" [fileDoc $ buildModule "VectorTest" []
+vectorTest = prog "VectorTest" "" "" [fileDoc $ buildModule "VectorTest" []
```

Comment:
Since we are using the `prog` function to define the `VectorTest`, we can manually input the version number but not automatically. If you want I can manually input the version number into it.

Reply:
Ah, gotcha. It's probably fine as is.

Reply:
Since we're going to try out the `drasil-metadata` approach, I think this is possible now.